smart/mesh.py

```diff
@@ -177,7 +177,14 @@ def __init__(
 
         self.child_meshes = dict()
         self.parent_mesh = self
-        self.curvature = curvature
+        if isinstance(curvature, (str, Path)) and Path(curvature).is_file():
```

Comment:
Is this double clause covering the case where `Curvature=""` ?

Reply:
Well, by default `curvature` is now set to `None`, which means that the double clause will cover the case when the user provides an invalid path.